libkodimote/episodes.cpp

```diff
@@ -179,9 +165,9 @@ void Episodes::listReceived(const QVariantMap &rsp)
         list.append(item);
         m_idIndexMapping.insert(item->episodeId(), index++);
     }
-    beginInsertRows(QModelIndex(), 0, list.count() - 1);
+    beginResetModel();
```

Comment:
This will cause the list to completely reload. as you're now updating on announcements (i.e. while the user is browsing the list) it would be better to merge the list and emit dataChanged signals instead.